tdrs-backend/tdpservice/parsers/util.py

```diff
@@ -284,13 +284,12 @@ def remove_case_due_to_errors(self, should_remove, case_hash):
 
 def generate_t1_t4_hashes(line, record):
     """Return hashes for duplicate and partial duplicate detection for T1 & T4 records."""
-    logger.debug(f"Partial Hash Field Values: {record.RecordType} {str(record.RPT_MONTH_YEAR)} {record.CASE_NUMBER}")
+    logger.debug(f"Partial Hash Field Values: for T1/T4: {record.RecordType} {str(record.RPT_MONTH_YEAR)} ")
     return hash(line), hash(record.RecordType + str(record.RPT_MONTH_YEAR or '') + str(record.CASE_NUMBER or ''))
```

Comment:
Can we keep CASE_NUMBER? I believe Alex indicated it isn't PII.

Reply:
in fact STTs sometimes assigns case numbers as SSN